src/test/ui/consts/const-eval/unwind-abort.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(unwind_attributes, const_panic)]
+
+#[unwind(aborts)]
+const fn foo() {
+    panic!() //~ evaluation of constant value failed
```

Comment:
Btw, shouldn't this say `//~ ERROR`? I don't think I have seen these comments without a message level before.

Reply:
If you'd like to ban this abbreviated form of test annotation, you're free to do so. I like them because they reduce noise.

Reply:
I'm not proposing to ban anything, I am asking if these are even checking anything, given that the docs do not mention them. If you like them, might be a good idea to add them to the docs.

Reply:
> I am asking if these are even checking anything

They are. They're used all over the test suite. I can document their existence I suppose, but I'm not sure why this responsibility falls on me.

Reply:
Fine then. I did a quick grep before asking you and I guess I didn't notice them in the vast, vast majority of annotations that to explicitly give a level. Given that they are also not documented, I think it is reasonable to ask.